README.md

```diff
@@ -10,7 +10,7 @@
 - Email: vuhn@pascaliaasia.com
 - Hobbies: Fishing
 
-# Ryo Asaka
-- DoB: September 23, 1994.
-- Email: ryo.asaka@g.pascalia.co.jp
-- Hobbies: Sports
+# Nobuyuki Ariga
+- DoB: September14 , 1997
+- Email: nobuyuki.ariga@g.pascalia.co.jp
+- Hobbies: Cyclingh
```

Comment:
As the diff, I see that you deleted Asaka-san's information and added your information. It is wrong.
Because you need to keep information of other members and add your information at bottom. Please fix it. Thanks.

Reply:
I understand. I'll fix it.